.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,32 @@
+{
+    "configurations": [
+        {
+            "name": "Mac",
+            "includePath": [
+                "${workspaceFolder}/common/**",
+                "${workspaceFolder}/editors/sc-ide/**",
+                "${workspaceFolder}/external_libraries/**",
+                "${workspaceFolder}/include/**",
+                "${workspaceFolder}/lang/**",
+                "${workspaceFolder}/platform/mac/**",
+                "${workspaceFolder}/QtCollider/**",
+                "${workspaceFolder}/SCDoc/**",
+                "${workspaceFolder}/server/**"
+            ],
+            "customConfigurationVariables": {
+                "-DCMAKE_PREFIX_PATH": "/usr/local/opt/qt@5",
+                "-DCMAKE_BUILD_TYPE": "Debug"
+            },
+            "defines": [],
+            "macFrameworkPath": [
+                "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks"
```

Comment:
Is it possible to install Xcode elsewhere? Perhaps if you had different versions? I'm not really a mac user.

Reply:
This was set via vscode itself and I think most devs have only one XCode version on their Mac as it is managed via the AppStore. I think it can be considered a sensible default, but maybe there is a better way of doing it, but I think the advanced devs with multiple XCode versions can figure out to modify this file in case they use vscode?

Reply:
This will probably cause problems if you don't have the Xcode app installed (and, for a vscode user at least, theres no reason to).

Reply:
But can you actually build w/o xcode? I thought you need XCode to obtain the clang compiler, so I think we can assume that anyone who wants to contribute to the C++ part of SuperCollider will also have to install XCode?